subprojects/android-test/junit-utils/src/main/java/com/avito/android/rule/SimpleRule.kt

```diff
@@ -10,15 +10,41 @@ abstract class SimpleRule : TestRule {
         object : Statement() {
             override fun evaluate() {
                 before()
-                try {
-                    base.evaluate()
-                } finally {
-                    after()
```

Comment:
The problem:
If test fails (`base.evaluate()`) and test rule fails (`after()`), test's exception will be lost.

Reply:
Additional note.
It's important to execute `after()` even in case of an error in tests. 
Custom Rules are counting on this behavior. They clean up a shared state.